samples/nrf9160/nrf_cloud_rest_device_message/src/main.c

```diff
@@ -398,6 +409,11 @@ void main(void)
 	rest_ctx.keep_alive = true;
 	(void)nrf_cloud_rest_alert_send(&rest_ctx, device_id,
 					ALERT_TYPE_DEVICE_NOW_ONLINE, 0, NULL);
+
+	err = nrf_cloud_rest_log_send(&rest_ctx, device_id, LOG_LEVEL_INF, "Hello World.");
```

Comment:
Maybe this could be replaced with the log message at the top of main:
```
LOG_INF("nRF Cloud REST Device Message Sample, version: %s",
		CONFIG_REST_DEVICE_MESSAGE_SAMPLE_VERSION);
```